docs/source/installation.md

```diff
@@ -0,0 +1,34 @@
+# Installation
+
+## With Docker
+
+First, download `vllm-spyre`:
+
+```shell
+git clone https://github.com/vllm-project/vllm-spyre.git
+cd vllm-spyre
+```
+
+Build image from source:
+
+```shell
+docker build . -f Dockerfile.spyre -t vllm-spyre
+docker run -it --rm vllm-spyre bash
```

Comment:
I think I'm still waiting on the PR from @ckadner to delete the Dockerfile!

Not sure if we want to document this path since we haven't used the dockerfile in months

Reply:
Removed this, keeping just the clone/uv install path.